sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -101,6 +104,16 @@ export interface AppleCriticalSound {
     volume: number;
 }
 
+// @public
+export interface AppleHeaders extends Record<string, any> {
```

Comment:
nit: `unknown` instead of `any` here?

Reply:
Moved to `Record<string, string | undefined>`